backend/TreeOfAKind.Application/Query/Trees/GetTree/TreeDto.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TreeOfAKind.Domain.Trees;
+using TreeOfAKind.Domain.Trees.People;
+
+namespace TreeOfAKind.Application.Query.Trees.GetTree
+{
+    public class PersonDto
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public Gender Gender { get; set; }
+        public DateTime? BirthDate { get; set; }
+        public DateTime? DeathDate { get; set; }
+        public string Description { get; set; }
+        public string Biography { get; set; }
+        public List<Guid> Mothers { get; set; } = new List<Guid>();
```

Comment:
`List` ???

Reply:
Yeah. Probably a little bit of an overkill. Was thinking of adoptive parents. Will remove

Reply:
We'll add lists when we finish other stuff